tests/test_adjoint_jacobian.py

```diff
@@ -26,9 +26,6 @@
 from pennylane import qchem, qnode
 from scipy.stats import unitary_group
 
-if device_name == "lightning.kokkos":
-    pytest.skip("Kokkos new API in WIP.  Skipping.", allow_module_level=True)
-
```

Comment:
All the unit / integration tests now are uncommented to check the functionality of the new device API with Kokkos